api/DHSC.FingertipsNext.Modules.HealthData.Repository/HealthDataRepository.cs

```diff
@@ -18,20 +17,23 @@ public class HealthDataRepository(HealthDataDbContext healthDataDbContext) : IHe
 
     public async Task<IndicatorDimensionModel> GetIndicatorDimensionAsync(int indicatorId)
     {
-        var results = await _dbContext.IndicatorDimension
-            .Where(i => i.IndicatorId == indicatorId)
-            .Select(x => new IndicatorDimensionModel()
-            {
-                IndicatorId = x.IndicatorId,
-                IndicatorKey = x.IndicatorKey,
-                Name = x.Name,
-                StartDate = x.StartDate,
-                EndDate = x.EndDate,
-                Polarity = x.Polarity,
-                BenchmarkComparisonMethod = x.BenchmarkComparisonMethod,
-            })
-            .ToListAsync();
-        return results.FirstOrDefault();
+        var model=await _dbContext.HealthMeasure
```

Comment:
```suggestion
        var mode l= await _dbContext.HealthMeasure
```

Minor but formatting. Don't think we have any linting imposed on the backend like we do on the frontend.

Reply:
done